pkg/aggregator/mocksender/asserts.go

```diff
@@ -46,6 +47,54 @@ func (m *MockSender) AssertMetricNotTaggedWith(t *testing.T, method string, metr
 	return m.Mock.AssertCalled(t, method, metric, mock.AnythingOfType("float64"), mock.AnythingOfType("string"), AssertTagsNotContains(tags))
 }
 
+// Compare an Event on specifics values
+func eventHaveEqualValues(expectedEvent, actualEvent metrics.Event) bool {
+	if assert.ObjectsAreEqualValues(expectedEvent.AggregationKey, actualEvent.AggregationKey) &&
+		assert.ObjectsAreEqualValues(expectedEvent.Priority, actualEvent.Priority) &&
+		assert.ObjectsAreEqualValues(expectedEvent.SourceTypeName, actualEvent.SourceTypeName) &&
+		assert.ObjectsAreEqualValues(expectedEvent.EventType, actualEvent.EventType) &&
+		expectedInActual(expectedEvent.Tags, actualEvent.Tags) {
+		return true
+	}
+	return false
+}
+
+// Iter on all mock.Calls to find any events who's matching the expectedEvent
```

Comment:
please keep the `$methodname...` format for docstrings even if vet won't complain for private methods